ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -741,11 +760,52 @@ const ControlledTierEraModule = (props: {
   ) : null;
 };
 
-const DARHeader = () => (
-  <FlexColumn style={styles.headerFlexColumn}>
-    <Header style={styles.headerRW}>Researcher Workbench</Header>
-    <Header style={styles.headerDAR}>Data Access Requirements</Header>
-  </FlexColumn>
+// the header outside the Fadebox
```

Comment:
not sure if it really makes style sense for two sets of headers, but I think these will make nice building blocks for iteration